lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -105,12 +106,18 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
         return (!this.hasFile || this.multipleOption) && !this.field.readOnly;
     }
 
-    onRemoveAttachFile(file: File | RelatedContentRepresentation | Node) {
+    onRemoveFile(file: File | RelatedContentRepresentation | Node) {
         this.removeFile(file);
         if (file['id'] === this.selectedNode?.id) {
             this.selectedNode = null;
-            this.contentModelFormFileHandler();
         }
+        if(file['id'] === this.lastSentToViewers?.id) {
+            this.contentModelFormFileHandler(this.field.value.length > 0 ? this.field.value[0] : null);
+        }
```

Comment:
Here's the actual fix - instead of letting the connected viewers know that a selected file was removed, we let them know that a file they're displaying has been removed

Reply:
Also - if there are still files in the list, display the first one. Otherwise, stop displaying.